python/ambassador/utils.py

```diff
@@ -169,7 +169,7 @@ def parse_bool(s: Optional[Union[str, bool]]) -> bool:
 
     # OK, we got _something_, so try strtobool.
     try:
-        return strtobool(s)
+        return bool(strtobool(s)) # the linter does not like a Literal[0, 1] being returned here
```

Comment:
:) I just ran into this one as well. The type narrowing from mypy doesn't seem to be working. is it an old version of mypy that we lint with?

Reply:
I think it's that we had `distutils` get updated. I just cherry-picked @AliceProxy's fix for this to include in the PR.